src/main/java/es/tid/pce/pcep/objects/tlvs/SRPolicyName.java

```diff
@@ -0,0 +1,119 @@
+package es.tid.pce.pcep.objects.tlvs;
+
+import java.util.Objects;
+
+import es.tid.pce.pcep.objects.MalformedPCEPObjectException;
+import es.tid.pce.pcep.objects.ObjectParameters;
+
+/*
+ * SR Policy Name TLV
```

Comment:
Add TVL Type code, a short description and the reference

Reply:
Done!